lib/token_util.py

```diff
@@ -26,6 +27,11 @@
 from glideinwms.lib.subprocessSupport import iexe_cmd
 
 
+def byt(x):
+    # return x.encode(defaults.BINARY_ENCODING)
+    return codecs.latin_1_encode(x)[0]
```

Comment:
I see that this method has been defined newly, but is this to be used/invoked somewhere in this script instead of what was existing? Or, is this just a helper method for some other use case because I do not see this method used at all.

Reply:
removed